woocommerce/woocommerce-ios
# [CPP Onboarding] Extract onboarding logic to separate file for better testability

Part of #4611 

This refactors the logic in #4689 and the tests from #4689 so that all the logic is in a separate `CardPresentPaymentsOnboardingUseCase` file instead of `CardPresentPaymentStore`, so it's easier to test.

## To test

Go to Settings > In-Person Payments:

1. With a store correctly set up for payments, you should see the option to connect to a reader
2. With a store that's not set up for payments, you should see a generic error

Note that this new implementation checks that the WCPay plugin is installed, but it doesn't manage refreshing the site plugins yet, so it might not work correctly if nothing else in the app has refreshed that data yet. I will look into that next.

Update release notes:
